ebib-utils.el

```diff
@@ -392,22 +392,33 @@ transformation function returns something that can be displayed."
   :type '(repeat (cons (string :tag "Field")
                        (function :tag "Transform function"))))
 
-(defcustom ebib-TeX-markup-replace-alist '(("{" . "")
-                                           ("}" . "")
-                                           ("``" . "“")
+(defcustom ebib-TeX-markup-replace-alist '(("``" . "“")
 					   ("`" . "‘")
 					   ("''" . "”")
-					   ("'" . "’"))
+					   ("'" . "’")
+					   ("\\\\textit\\(?:\\[.*\\]\\)?{\\(.*?\\)}"
```

Comment:
AFAIK `textit` et. al. don't accept optional arguments (the LaTeX2e author manual doesn't mention them) so these regexes can be simplified to `"\\\\textit{\\(.*?\\)}"`.

Reply:
Fixed in most recent push.